cyberark-secrets-provider-for-kubernetes/values.yaml

```diff
@@ -0,0 +1,36 @@
+# K8s permission defaults
+rbac:
+  roleName: secrets-provider-role
+  roleBindingName: secrets-provider-role-binding
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Name of the service account to use
+    name: secrets-provider-service-account
+
+secretsProvider:
+  image: cyberark/secrets-provider-for-k8s
+  tag: 1.0.0
+  name: cyberark-secrets-provider
+
+## Mandatory values to be filled in by customer
+## Examples values given in https://docs.conjur.org/<PATH TBD>
+environment:
+  # Comma-separated list of Kubernetes Secret names that applications consume and their value is sourced in DAP/Conjur
+  # This key should be in array format. For example, [k8s-secret1,k8s-secret2]
+  k8sSecrets:
+  conjur:
+    # Account name designated to the Conjur/DAP during initial configuration
+    account:
+    # URL of service defined for Conjur/DAP
+    applianceUrl:
+    # Credential service used to log into Conjur/DAP
```

Comment:
What do you think about adding an example in the comments? I know that we have JSON schema validation, but would be nice for a user to know how does an expected input look like, before it fails in the validation.

Reply:
We also plan to write documentation with examples. Do you think that that will be sufficient or do you still feel examples in the comments give a better experience for the user?

Reply:
I think documentation is better, because values.yaml arrives to the customer in many cases inside hel.tgz file and hence the customer will not see it

Reply:
I disagree. The `values.yaml` is the file that the end user edits to customize the Helm chart values for his/her setup. If the user will look at any file in the Helm chart, this would be the one.

Reply:
this file is called usually custom-values.yaml @rafis3 
The file that is called values.yaml is part of helmchart itself.

Reply:
following to our offline conversation with Rafi, we advice that the required format of input will be displayed as an output of helm install. Maybe it os done automatically by json schema mechanism. @sigalsax please check it - and if it is not - it must be added explicitly to the output